packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import DotcomShell from '../DotcomShell';
+import {
+  Button,
+  DataTableSkeleton,
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListCell,
+} from 'carbon-components-react';
+import mastheadLinks from '../../Masthead/__stories__/data/MastheadLinks.js';
+import '@ibmdotcom/styles/scss/components/dotcom-shell/_dotcom-shell.scss';
```

Comment:
Only minor thing, I think for our stories we can do relative path to our styles package so we will get the styles that aren't pushed up to npm.